ReactWindows/ReactNative/ReactNative.csproj

```diff
@@ -129,44 +129,14 @@
     <Compile Include="Animated\StyleAnimatedNode.cs" />
     <Compile Include="Animated\TransformAnimatedNode.cs" />
     <Compile Include="Animated\ValueAnimatedNode.cs" />
-    <Compile Include="Bridge\BridgeBusyEventArgs.cs" />
-    <Compile Include="Bridge\BridgeIdleEventArgs.cs" />
-    <Compile Include="Bridge\ILifecycleEventListener.cs" />
-    <Compile Include="Bridge\IPromise.cs" />
     <Compile Include="Bridge\ReactInstance.cs" />
-    <Compile Include="Bridge\CompiledReactDelegateFactory.cs" />
     <Compile Include="Bridge\DispatcherHelpers.cs" />
-    <Compile Include="Bridge\IInvocationHandler.cs" />
-    <Compile Include="Bridge\IJavaScriptExecutor.cs" />
-    <Compile Include="Bridge\IJavaScriptModule.cs" />
-    <Compile Include="Bridge\IReactCallback.cs" />
-    <Compile Include="Bridge\IReactDelegateFactory.cs" />
-    <Compile Include="Bridge\JavaScriptModuleBase.cs" />
-    <Compile Include="Bridge\JavaScriptModuleBase.Generated.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>JavaScriptModuleBase.Generated.tt</DependentUpon>
-    </Compile>
-    <Compile Include="Bridge\JavaScriptModuleRegistration.cs" />
-    <Compile Include="Bridge\JavaScriptModuleRegistry.cs" />
     <Compile Include="Bridge\JavaScriptBundleLoader.cs" />
```

Comment:
They have UWP specifics that were not immediately obvious on how to convert them. I'll need some help figuring out how to transliterate those into PCL-compatible code (if its even possible), but also I think the platform-specific projects implementing concrete types for  subclasses/interfaces in the PCL projects is a legit way to split things.